common/djangoapps/student/views/management.py

```diff
@@ -542,6 +547,10 @@ def activate_account(request, key):
                 extra_tags='account-activation aa-icon',
             )
 
+    if should_redirect_to_logistration_mircrofrontend() and not request.user.is_authenticated:
+        query_param = '?activate_account={}'.format(activation_message_type)
+        return redirect(reverse('dashboard') + query_param)
```

Comment:
Don't we need to redirect to the logistration MFE login page here instead of the dashboard?

Reply:
It will go to dashboard and on dashboard it automatically redirects to login page (FE or MFE appropriately) 
I didn't want to change the existing behaviour of redirecting to dashboard 
https://github.com/edx/edx-platform/blob/41dc0b25a4f8bf8291add440c042d4ae9329ec25/common/djangoapps/student/views/management.py#L554

Reply:
I guess the query params will be lost during that redirection?

Reply:
And I think we should avoid that extra redirection if we already know the user is unauthenticated.

Reply:
sure can do that